test/hotspot/jtreg/applications/scimark/Scimark.java

```diff
@@ -47,7 +46,9 @@ public static void main(String... args) throws Exception {
                             + Scimark.class.getName(), e);
         }
 
-        OutputAnalyzer output = new OutputAnalyzer(ProcessTools.createJavaProcessBuilder(
+        System.setProperty("test.noclasspath", "true");
```

Comment:
What does this do?

Reply:
The "test.noclasspath" is set to don't use the default classpath to run process and only add VM options. This test starts the 3rd-party application and doesn't need test classes and a test library in the path.